### DISCUSSION TITLE
support pnpm

### DISCUSSION THREAD

**Discussion by SvenMeyer** [scaffold-eth/scaffold-eth-2#624] [Ideas]
Would be great if it would be possible to use pnpm instead of yarn.
At least it did not work for me out of the box ...
```
$ pnpm chain

> se-2@0.0.1 chain /home/sum/DEV/ETH/WebUI/scaffold-eth-2-test-1
> yarn workspace @se-2/hardhat chain

Usage Error: Couldn't find the node_modules state file - running an install might help (findPackageLocation)

$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...
 ELIFECYCLE  Command failed with exit code 1.

```

**Comment by technophile-04** [comment]
Hey thanks for this 🙌, check out this discussion in PR #444 and this comment https://github.com/scaffold-eth/scaffold-eth-2/pull/544#issuecomment-1737451534. 

TLDR; currently as such there is no plan to move to `pnpm` but hopefully in future there is a plan to support / give option to pick `pnpm`, bun etc when using SE-2 through `npx create-eth@latest`
